interactions/client/bot.py

```diff
@@ -135,7 +135,10 @@ def latency(self) -> float:
 
     def start(self) -> None:
         """Starts the client session."""
-        self._loop.run_until_complete(self._ready())
+        try:
+            self._loop.run_until_complete(self._ready())
+        except (CancelledError, Exception) as e:
```

Comment:
Why `Exception`?

Reply:
To make the exception scope broader. Do you think it's not needed? I was questioning it as well

Reply:
I don't know xD that's why I asked you... I'm just approving it, it's not hurting anyone if it is there